### PR TITLE
TYPO3 V8 Updates

### PR DESCRIPTION
**Pull request by TomSchenk**
Just for Code Review reasons - do not merge!

### CODE REVIEW

File: Classes/Realurl.php [hunk 1]
```diff
@@ -193,7 +202,7 @@ public function __construct()
         $this->enableStrictMode = (bool) $sysconf['enableStrictMode'];
         $this->enableChashUrlDebug = (bool) $sysconf['enableChashUrlDebug'];
         $this->enableRootlineExceptionLog = (bool) $sysconf['enableRootlineExceptionLog'];
-        $this->configurationService = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('tx_realurl_configurationService');
+        $this->configurationService = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(ConfigurationService::class);
```
Comment (felixsemmler) [inline Classes/Realurl.php]: Remove fully qualified namespace, GeneralUtility is already imported.

File: Classes/Tcemain.php [hunk 2]
```diff
@@ -379,13 +383,13 @@ protected function shouldFixCaches($tableName, array $databaseData)
     protected function markCachesDirty($tableName, $recordId, &$reference)
     {
         if ($tableName == 'pages') {
-            $cache = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('tx_realurl_cachemgmt', $GLOBALS ['BE_USER']->workspace, 0);
+            $cache = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(Cachemgmt::class, $GLOBALS ['BE_USER']->workspace, 0);
```
Comment (felixsemmler) [inline Classes/Tcemain.php]: Add use statement instead of fully qualified namespace?

File: modfunc1/Modfunc1.php [hunk 3]
```diff
@@ -1,4 +1,6 @@
 <?php
+namespace AOE\Realurl\modfunc1;
```
Comment (felixsemmler) [inline modfunc1/Modfunc1.php]: Lowercase namespace for modfunc1? Is this ok?

File: Classes/Tcemain.php (hunk 2 above)
Comment (aoekrz) [reply Classes/Tcemain.php]: Done

File: Classes/Realurl.php (hunk 1 above)
Comment (aoekrz) [reply Classes/Realurl.php]: Done

File: modfunc1/Modfunc1.php (hunk 3 above)
Comment (aoekrz) [reply modfunc1/Modfunc1.php]: Done

**Comment by TomSchenk** [comment]
Just for Code Review reasons - Code Review done

**Pull request closed without merging**